### PR TITLE
DAT-3788 add default mercury spinner when saving infobox builder

### PR DESCRIPTION
**Pull request by dianafa**
## Links

* https://wikia-inc.atlassian.net/browse/DAT-3788

## Description

Add spinner on save action to infobox builder.

## Reviewers

@RafLeszczynski @rogatty @hakubo

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: front/main/app/templates/components/infobox-builder.hbs [hunk 1]
```diff
@@ -50,3 +49,24 @@
 		editRowItem=editRowItem
 	}}
 </div>
+{{#if showOverlay}}
+	<div class="infobox-builder-spinner loading-overlay">
```
Comment (RafLeszczynski) [inline front/main/app/templates/components/infobox-builder.hbs]: what do you think about making this a reusable component ?

File: front/main/app/templates/components/infobox-builder.hbs [hunk 2]
```diff
@@ -50,3 +49,24 @@
...
+			radius=38
+			overlay=false
+			strokeWidth=3
+		}}
+		<div class="spinner-text">
+			{{i18n 'main.saving' ns='infobox-builder'}}
+		</div>
+	{{/if}}
+	{{#if showSuccess}}
+		{{svg 'success' class='success-icon' viewBox='0 0 80 80'}}
```
Comment (RafLeszczynski) [inline front/main/app/templates/components/infobox-builder.hbs]: same as above

File: front/main/app/styles/component/_infobox-builder.scss [hunk 3]
```diff
@@ -290,3 +292,29 @@ $grayout-color: rgba(255,255,255,0.3);
 		}
 	}
 }
+
+.infobox-builder-spinner {
```
Comment (RafLeszczynski) [inline front/main/app/styles/component/_infobox-builder.scss]: if you decide to go with separate component move the spinner specific styles to separate scss file

**Pull request merged**